app/services/external_data/pokemon/tcg/players.rb

```diff
@@ -0,0 +1,62 @@
+module ExternalData
+
+  module Pokemon
+
+    module Tcg
+
+      class Players
+
+        include HTTParty
+        base_uri 'https://limitlesstcg.com'
+
+        def self.all
+          get_from(players: [], page: 1)
+        end
+
+        def self.get_players_by(region: 'all', time: 'all', game_format: 'all', show: 100, page: 1)
+          get('/players', query: { region:, time:, format: game_format, type: 'all', show:, page: })
+        end
+
+        def self.get_from(players: [], page: 1)
```

Comment:
i'd rename this method, it sounds to me like it fetches all records from a certain page, while it actually fetches all records from that page and up, if I read it correctly

Reply:
Yeah, naming here wasn't convenient. In essence this is the recursive method we need to run the parser. I'll give it another shot